templates/install_puppet.bat.erb

```diff
@@ -0,0 +1,11 @@
+:wait_for_pid
+FOR /F "tokens=*" %%A IN ('tasklist /FI "PID eq <%= @puppet_agent_pid %>"  /NH') DO set _task=%%A
+echo %_task% | findstr "No tasks are running" >nul
+IF NOT %errorlevel% == 0 ( GOTO wait_for_pid )
+
+
+msiexec.exe /qn /i <%= @_msi_location %> /l*v <%= @_logfile %>
```

Comment:
We are not creating a secondary log, to what point would we need to do a start /w? Will add /norestart but I have not seen it needed in any of the tests and it is extremely unlikely to require it.